bip39appv1_7.py

```diff
@@ -37,14 +37,43 @@
 
 
 def main():
-    entropy = generate_entropy()
+    word_count = ask_word_count()
+    checksum_bit_count = word_count // 3
```

Comment:
Why is this way (dividing wordcount by 3) of computing the length of the checksum better than by dividing the initial entropy by 32?

Reply:
Let me see. Basically, we don't know entropy from the start, right? What we have is word count, taken from the input. Is there any way to count entropy before counting number of bits in checksum? If there is, we can use it instead and then calc checksum bits by division to 32.

Reply:
I think both options achieve the same result so not a big deal. Regarding your question, if a user chooses 12 then the resulting 128 bits that are either user-supplied or randomly generated could be divided by 32 to compute the checksum, I guess it's just another way to calculate but I am not seeing any difference in any performance gains either way. Thanks